pkgs/development/compilers/llvm/6/compiler-rt.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, version, fetch, cmake, python, llvm }:
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "compiler-rt-${version}";
+  inherit version;
+  src = fetch "compiler-rt" "16m7rvh3w6vq10iwkjrr1nn293djld3xm62l5zasisaprx117k6h";
+
+  nativeBuildInputs = [ cmake python llvm ];
```

Comment:
Pretty sure LLVM is just a build tool here? So it is a `nativeBuildInput`.

Reply:
Reallllyyy??? No wonder you were asking about using a stand-in for llvm-config, wonder what the details here are.